cmd/generate/BuildImageDockerfile.template

```diff
@@ -18,6 +18,7 @@ RUN wget https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 &&
 RUN ./get-helm-3 --version v3.11.3 --no-sudo && helm version
 
 RUN GOFLAGS='' go install github.com/mikefarah/yq/v3@latest
```

Comment:
`yq` is required by S-O generators, but there's script to install it per use, AFAIK.

Reply:
https://github.com/openshift-knative/serverless-operator/blob/main/hack/lib/metadata.bash#L3-L13

Reply:
I think we use it unconditionally in Eventing

Reply:
OK